backends/rapidpro/status.go

```diff
@@ -341,3 +350,42 @@ func writeMsgStatuses(ctx context.Context, db *sqlx.DB, spoolDir string, statuse
 		}
 	}
 }
+
+const sqlResolveStatusMsgIDs = `
+SELECT id, channel_id, external_id 
+  FROM msgs_msg 
+ WHERE (channel_id, external_id) IN (VALUES(CAST(:channel_id AS int), :external_id))`
```

Comment:
note that you can't write `:channel_id::int` because sqlx will get confused booo